File: src/main/resources/output/IFC4X3_js/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcCurve/IfcSurfaceCurve/IfcSeamCurve/IfcSeamCurve.js
class IfcSeamCurve extends IfcSurfaceCurve {
    constructor() {
        // no parameters
    }

    // === WHERE CLAUSES ===
    // SameSurface : IfcAssociatedSurface(SELF\IfcSurfaceCurve.AssociatedGeometry[1]) = IfcAssociatedSurface(SELF\IfcSurfaceCurve.AssociatedGeometry[2])
    // TwoPCurves : SIZEOF(SELF\IfcSurfaceCurve.AssociatedGeometry) = 2
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcCurve/IfcSurfaceCurve/IfcSeamCurve/IfcSeamCurve.vb
Public Class IfcSeamCurve Inherits IfcSurfaceCurve

    ' === WHERE CLAUSES ===
    ' SameSurface : IfcAssociatedSurface(SELF\IfcSurfaceCurve.AssociatedGeometry[1]) = IfcAssociatedSurface(SELF\IfcSurfaceCurve.AssociatedGeometry[2])
    ' TwoPCurves : SIZEOF(SELF\IfcSurfaceCurve.AssociatedGeometry) = 2
End Class
